watertap/unit_models/zero_order/electrocoagulation_zo.py

```diff
@@ -288,43 +262,95 @@ def build(self):
             doc="Power required",
         )
 
+        # Flocculator Variables
+
+        self.floc_basin_vol = Var(
+            initialize=1,
+            bounds=(0, None),
+            units=pyunits.m**3,
+            doc="Reactor volume total (flotation + sedimentation)",
+        )
+
+        self.floc_retention_time = Var(
+            initialize=30,
+            bounds=(2, 200),
+            units=pyunits.minute,
+            doc="Electrolysis time",
+        )
+
         self._fixed_perf_vars.append(self.electrode_thick)
         self._fixed_perf_vars.append(self.current_density)
-        self._fixed_perf_vars.append(self.electrolysis_time)
-        self._fixed_perf_vars.append(self.metal_loading)
-        self._fixed_perf_vars.append(self.number_electrode_pairs)
+        self._fixed_perf_vars.append(self.metal_dose)
+        self._fixed_perf_vars.append(self.conductivity)
         self._fixed_perf_vars.append(self.electrode_gap)
         self._fixed_perf_vars.append(self.current_efficiency)
-        self._fixed_perf_vars.append(self.overpotential)
+        self._fixed_perf_vars.append(self.floc_retention_time)
+
+        if self.config.overpotential_calculation is OverpotentialCalculation.fixed:
+            self._fixed_perf_vars.append(self.overpotential)
+
+        if self.config.overpotential_calculation == OverpotentialCalculation.calculated:
```

Comment:
Pretty easy to get this config option covered by tests? I'm not sure how particular the ZO unit models are to be.

Reply:
Good catch, just added a test.